src/pages/og/[...slug].svg.ts

```
@@ -0,0 +1,99 @@
+      <rect x="60" y="36" width="48" height="48" rx="8" fill="#3B82F6"/>
+      <text x="84" y="66" text-anchor="middle" class="category">C</text>
+      
+      <!-- Brand -->
+      <text x="128" y="70" class="category" style="font-size: 28px;">Cor.inc</text>
+      
+      <!-- Category -->
+      <rect x="1020" y="45" width="120" height="30" rx="15" fill="#3B82F6"/>
+      <text x="1080" y="65" text-anchor="middle" class="category">${category.toUpperCase()}</text>
+      
+      <!-- Title -->
+      <text x="60" y="220" class="title">${safeTitle.length > 50 ? safeTitle.substring(0, 50) + '...' : safeTitle}</text>
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)